pyproject.toml

```diff
@@ -14,10 +14,18 @@ classifiers = [
     "Development Status :: 4 - Beta",
     "Environment :: Web Environment",
     "Framework :: Django",
-    "Framework :: Django :: 2.2",
     "Framework :: Django :: 3.2",
+    "Framework :: Django :: 4.1",
+    "Framework :: Django :: 4.2",
     "Intended Audience :: Developers",
+    "License :: OSI Approved :: BSD License",
     "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
```

Comment:
Note that the Python version classifiers are already specified for [Kagi on PyPI](https://pypi.org/project/kagi/). I don't know how it works with other packaging tools such as Hatchling/PDM/Hatch, but with Poetry there is no need to explicitly maintain this list of classifiers, as that is handled automatically by inspecting the supported Python versions (in this case, `python = ">= 3.7, < 4.0"`).

Personally, I find it liberating to avoid the need to maintain this list manually. 😊

Reply:
Oh, interesting. I keep that in mind next time :)